docs/src/modules/components/AppFrame.js

```diff
@@ -235,7 +235,12 @@ export default function AppFrame(props) {
 }
 
 AppFrame.propTypes = {
+  BannerComponent: PropTypes.elementType,
   children: PropTypes.node.isRequired,
   className: PropTypes.string,
   disableDrawer: PropTypes.bool,
 };
+
+AppFrame.defaultProps = {
+  BannerComponent: AppFrameBanner,
```

Comment:
Should we set `enable_docsnav_banner` to `false` to hide the current banner given the upcoming v6 stable release?

Reply:
Yes, that's a good point. By the way, do you think it will improve adaption a bit by keep showing this banner (after updating text `v6-beta` -> `v6`) on core docs for a few weeks after stable? After a short period we can remove it from core docs and only keep it in v5 X-specific docs. What do you think?

Something like:
<img width="607" alt="image" src="https://user-images.githubusercontent.com/12609561/220816373-cc242cc2-4fb9-42e3-8011-f46001a5dffe.png">

Side note: I noticed that this banner is not displayed for smaller width screens i.e. `< 1200px` (when the sidebar disappears), this is probably to keep UI for mobile screens consistent and single liner but I feel some desktop displays also fall under the same screen resolution range.

I am not sure how much a percentage of users are on smaller displays, but adding this on smaller ones will possibly improve its visibility a bit.

Reply:
> do you think it will improve adaption a bit by keep showing this banner (after updating text v6-beta -> v6) on core docs for a few weeks after stable?

That's a good idea for sure!
Although we have to make sure that the v6 stable banner only appears after the release is done.
We can keep the current banner, and then update it as soon as the v6 stable is available and deploy it to production (new instructions in https://github.com/mui/material-ui/pull/36301 could be helpful for the deployment process)

> I am not sure how much a percentage of users are on smaller displays

In the last 12 months, the breakdown looks like this:
- desktop: 97.73%
- mobile: 2.19%
- tablet: 0.09%

https://analytics.google.com/analytics/web/#/report/visitors-mobile-overview/a106598593w187396170p184180857/_u.date00=20220223&_u.date01=20230223

I would not focus too much on adding this banner on mobile.